modules/contact/src/problems/AugmentedLagrangianContactProblem.C

```diff
@@ -26,26 +26,12 @@
 #include "ConstraintWarehouse.h"
 #include "MortarUserObject.h"
 #include "AugmentedLagrangeInterface.h"
+#include "AugmentedLagrangianContactConvergence.h"
+#include "Convergence.h"
 
 registerMooseObject("ContactApp", AugmentedLagrangianContactProblem);
 registerMooseObject("ContactApp", AugmentedLagrangianContactFEProblem);
 
-InputParameters
-AugmentedLagrangianContactProblemInterface::validParams()
-{
-  auto params = emptyInputParameters();
-  params.addParam<int>("maximum_lagrangian_update_iterations",
-                       100,
-                       "Maximum number of update Lagrangian Multiplier iterations per step");
-  return params;
-}
-
-AugmentedLagrangianContactProblemInterface::AugmentedLagrangianContactProblemInterface(
-    const InputParameters & params)
-  : _maximum_number_lagrangian_iterations(params.get<int>("maximum_lagrangian_update_iterations"))
-{
-}
```

Comment:
I see this isnt your or Oana's design

Reply:
Is this saying "nevermind" to your previous comment?

Reply:
see the other nevermind comment. no action to do here